src/Core/src/Fonts/EmbeddedFontLoader.iOS.cs

```diff
@@ -1,25 +1,15 @@
+#nullable enable
 using System;
-using System.Diagnostics;
 using CoreGraphics;
 using CoreText;
 using Foundation;
+using Microsoft.Extensions.Logging;
 using UIKit;
 
 namespace Microsoft.Maui
 {
-	public class EmbeddedFontLoader : IEmbeddedFontLoader
+	public partial class EmbeddedFontLoader
 	{
-
-#if !NET6_0
-		// The NET6_0 linker won't need this
-		// Make sure to test with full linking on before removing
-		[Preserve]
```

Comment:
Did you test removing this against the Control Gallery on iOS? And turning on the full linker experience?

Reply:
No 😅 But I added a second ctor and is now directly referenced in the font registration extension file:

```csharp
services.AddSingleton<IEmbeddedFontLoader>(svc => new EmbeddedFontLoader(svc.CreateLogger<EmbeddedFontLoader>()));
```

Reply:
Also, we are only building for net6 out there, so no classic code needed I think... Should I add it back?

Reply:
it's mainly relevant for the Control Galleries when running UI Tests in the CI
So if we get the Control Galleries converted to NET6 and run those in the galleries than no
But we might want to add it back until then

Reply:
Done.